src/App.js

```diff
@@ -1,16 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import ColorChoice from './components/ColorChoice';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
+
+  const SENDER_1 = chatMessages[0].sender;
+  const SENDER_2 = chatMessages[1].sender;
```

Comment:
This happens to work for the example data. Think about how you might handle it if the data started with a few messages from the same sender!